Content.Shared/Interfaces/IModuleManager.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Content.Shared.Interfaces
+{
+    /// <summary>
+    /// Provides a simple way to check whether calling code is being run by
+    /// Robust.Client, or Robust.Server. Useful for code in Content.Shared
+    /// that wants different behavior depending on if client or server is using it.
+    /// </summary>
+    public interface IModuleManager
+    {
+        /// <summary>
+        /// Returns true if the code is being run by the client, returns false otherwise.
+        /// </summary>
+        bool IsClientModule();
```

Comment:
Make this a property?

Reply:
Done. Didn't know you could have properties in interfaces. Thought it was functions only.

Reply:
Properties are for all intents and purposes just functions with some syntax sugar, so yeah.